src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
+      >
+        <DrawerHeader>
+          <IconButton onClick={toggleDrawer}>
+            {!open && hover && !expanded && (
+              <KeyboardDoubleArrowRightIcon
+                onClick={toggleDrawer}
+                sx={{ margin: 1 }}
+              />
+            )}
+            {!open && !hover && !expanded && (
+              <NextLink href="/organize" passHref>
+                <Avatar
+                  alt="icon"
+                  src={`/api/orgs/${orgId}/avatar`}
+                  sx={{ alignSelf: 'center' }}
+                />
+              </NextLink>
+            )}
+            {expanded && (
+              <ZUIFuture future={model.getOrganization(orgId)}>
+                {(data) => {
+                  return (
+                    <Box alignSelf="center">
+                      <Box display="flex" justifyContent="start">
+                        {hover ? (
+                          <ChevronLeftIcon
+                            onClick={handleDrawerToggle}
+                            sx={{ alignSelf: 'center', margin: 2 }}
+                          />
+                        ) : (
+                          <Avatar
+                            alt="icon"
+                            src={`/api/orgs/${orgId}/avatar`}
+                            sx={{ alignSelf: 'center', marginLeft: 2 }}
+                          />
+                        )}
+                        <Typography gutterBottom m={2} variant="h6">
+                          {data.title}
+                        </Typography>
+                      </Box>
+                    </Box>
+                  );
+                }}
+              </ZUIFuture>
+            )}
+          </IconButton>
+        </DrawerHeader>
+        <Divider />
+        <List>
+          {menuItemsMap.map((item) => (
+            <ListItem
+              key={item.name}
+              disableGutters
+              disablePadding
+              sx={{ display: 'block' }}
+            >
+              <NextLink
+                href={
+                  item.name === 'home'
+                    ? '/organize/'
+                    : `/organize/${orgId}/${item.name}`
+                }
+                passHref
               >
-                <Event />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-        </Box>
-      </List>
+                <IconButton
+                  color={
+                    key.startsWith('/' + item.name) ? 'primary' : 'secondary'
+                  }
+                  sx={{
+                    justifyContent: open ? 'initial' : 'center',
+                    minHeight: 48,
+                    px: 2.5,
+                  }}
+                >
+                  <ListItemIcon
+                    sx={{
+                      justifyContent: 'center',
+                      minWidth: 0,
+                      mr: open ? 3 : 'auto',
+                    }}
+                  />
+                  {item.icon}
```

Comment:
This looks weird to me. Should the `item.icon` be inside `<ListItemIcon>`? Or can `ListItemIcon` be removed?

Reply:
I've removed ListItemIcon!